standard_template/standard/clause_specification_text.adoc

```diff
@@ -965,6 +965,7 @@ Coverage example:
 ### 10.2. VerticalProfile
 
 - A domain with VerticalProfile domain type MUST have the axes `"x"`, `"y"`, and `"z"`, where `"x"` and `"y"` MUST have a single coordinate only.
+- A domain with VerticalProfile domain type MAY have the axis `"t"` which MUST have a single coordinate only.
```

Comment:
would it be more clear to state that the `t` axis must have a single coordinate value only?

Reply:
yes, add the word value to make it slightly clearer